frontend-ssr/redux/exercise-submissions/exercise-submissions.actions.ts

```diff
@@ -30,3 +36,21 @@ export const updateSubmission = (id: string, newSubmission: Submission) => ({
     newSubmission,
   },
 });
+
+export const loadSubmissions = (n: number) => ({
+  type: LOAD,
+  payload: {
+    n,
+  },
+});
+
+export const searchSubmissions = (keyword: string) => ({
+  type: SEARCH,
+  payload: {
+    keyword,
+  },
+});
+
+export const nextPage = () => ({
```

Comment:
No need for `nextPage` action or action creator for now. We'll need it in step 3 when we add some async work